docker/tangle-parachain/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+# Docker compose file to start tangle parachain collator that connects to tangle testnet
+
+version: "3.7"
+services:
+
+  tangle:
+    image: ghcr.io/webb-tools/tangle/tangle-parachain:${RELEASE_VERSION}
+    network_mode: host
+    entrypoint: /tangle-parachain
+    command:
+      - "--chain"
+      - "/var/lib/data/arana-alpha.json"
+      - "--validator"
+      - "--unsafe-ws-external"
+      - "--unsafe-rpc-external"
```

Comment:
We should not have these flags included here if we are using it with the `validator` flag.

Reply:
Ya i added it to make key insertion easier, I guess we can remove them